extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts

```diff
@@ -479,22 +480,51 @@ export class MigrationStatusDialog {
 			.component();
 
 		if (count > 0) {
-			control.addItems([
-				// migration warning / error image
-				this._view.modelBuilder.image().withProps({
+			const migrationWarningImage = this._view.modelBuilder.image()
+				.withProps({
 					iconPath: this._statusInfoMap(status),
 					iconHeight: statusImageSize,
 					iconWidth: statusImageSize,
 					height: statusImageSize,
 					width: statusImageSize,
 					CSSStyles: imageCellStyles
-				}).component(),
-				// migration warning / error counts
-				this._view.modelBuilder.text().withProps({
-					value: loc.STATUS_WARNING_COUNT(status, count),
+				}).component();
+
+			const migrationWarningCount = this._view.modelBuilder.hyperlink()
+				.withProps({
+					label: loc.STATUS_WARNING_COUNT(status, count) ?? '',
+					url: '',
 					height: statusImageSize,
 					CSSStyles: statusCellStyles,
-				}).component()
+				}).component();
+			this._disposables.push(migrationWarningCount.onDidClick(async () => {
+				clearDialogMessage(this._dialogObject);
+				const cutoverDialogModel = new MigrationCutoverDialogModel(migration!);
+				const errors = await cutoverDialogModel.fetchErrors();
+				this._dialogObject.message = {
+					text: errors
+						.filter((e, i, arr) => e !== undefined && i === arr.indexOf(e))
+						.join(EOL),
+					level: status === MigrationStatus.InProgress
+						|| status === MigrationStatus.Completing
+						? azdata.window.MessageLevel.Warning
+						: azdata.window.MessageLevel.Error,
+					description: (cutoverDialogModel.migrationOpStatus) ?
+						(JSON.stringify(
+							{
+								'async-operation-details': cutoverDialogModel.migrationOpStatus,
+								'details': status
+							}
+							, undefined, 2)) :
+						(JSON.stringify(status, undefined, 2))
+				};
+			}));
+
+			control.addItems([
+				// migration warning / error image
+				migrationWarningImage,
+				// migration warning / error counts
```

Comment:
let's remove the comments now. The variable name is now clear enough

Reply:
Done. Thanks